cli/fossilize_replay.cpp

```diff
@@ -2776,6 +2785,8 @@ struct ThreadedReplayer : StateCreatorInterface
 									 // if we keep them around indefinitely.
 					                 free_pipelines();
 
+					                 parents.clear();
```

Comment:
Isn't this cleared elsewhere already?

Reply:
I moved it here because `parents` is used by `ENQUEUE_DERIVED_PIPELINES_OFFSET`.